public/styles/main.css

```diff
@@ -251,3 +281,92 @@ a.connections-table.icon {
   padding: 0 0.5rem;
   font-size: 1rem;
 }
+
+.new-connection-input-field {
+  min-width: 200px;
+  max-width: 230px;
+  padding: 10px;
+  border-radius: 7px;
+  margin: 10px;
+}
+.align-horizontally {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+}
+.align-company-information {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  padding: 1rem;
+  border: 2px solid green;
+  border-radius: 7px;
+}
+.align-buttons-left {
+  display: flex;
+  min-width: 200px;
+  max-width: 40%;
+  align-items: left;
+  padding-left: 10px;
+}
+.align-buttons-left button {
+  margin-left: 10px;
+}
+.direction-column {
+  display: flex;
+  flex-direction: column;
+}
+.flex-form-content {
+  display: flex;
+  justify-content: left;
+  align-items: left;
+}
+#text-box-target {
+  border: 40px var(--text-color);
+
+  border-radius: 12px;
+  padding: 10px 50px 10px 20px;
+}
+.centered {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+.progress-container {
+  display: flex;
+  align-items: center;
+  width: 90%;
+  margin: 20px auto;
+}
+
+.progress-bar {
+  flex: 1;
+  height: 5px;
+  background-color: #ddd;
+  position: relative;
+  margin-right: 10px;
+}
+
+.progress-bar::before {
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100%;
+  background-color: var(--accent-color);
+}
+.bar-1-3::before {
+  width: 33.33%; /* 1/3 of the bar */
+}
+.bar-2-3::before {
+  width: 66.66%; /* 2/3 of the bar */
+}
+.bar-3-3::before {
+  width: 100%; /* 3/3 of the bar */
+}
```

Comment:
one really small thing, it's up to you if you change. Can you change these to instead of changing width making them change the `transform` property with https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/scaleX? This will make it smoother when we animate this (which I can definitely see us doing). See also https://developer.mozilla.org/en-US/docs/Web/CSS/transform-origin